pcurve/__init__.py

```diff
@@ -0,0 +1,2 @@
+from pcurve.pcurve import *
+from pcurve.project_to_curve_converted import *
```

Comment:
`project_to_curve_converted` has been deleted in `master`, can you delete it here too? (It is not used anywhere).